src/views/Main/Login/Login.js

```diff
@@ -1,7 +1,7 @@
 import React, { PropTypes } from 'react';
 import { ButtonToolbar, Button } from 'react-bootstrap';
 import AuthService from '../../../utils/AuthService';
-import styles from './styles.module.css';
+import './styles.module.css';
```

Comment:
we need to double check how this works, but I think that this `import` tells webpack to create a `<link>` element in the HTML to inject the CSS, so you don't have to import the styles in each module. you'll probably have to import the style once in `app.js`.